location1: {Confidence: 100, x: 1.71, y: -1.06}
location2: {Confidence: 100, x: -1.13, y: 1.5}
location3: {Confidence: 100, x: -5.06, y: 1.5}
location4: {Confidence: 0, x: 0.0, y: 0.0}
